cmd/api/src/model/unified_graph.go

```diff
@@ -78,6 +79,7 @@ func FromDAWGSNode(node *graph.Node, includeProperties bool) UnifiedNode {
 		Kind:       analysis.GetNodeKind(node).String(),
 		ObjectId:   objectId,
 		IsTierZero: strings.Contains(systemTags, ad.AdminTierZero),
+		IsOwnedObject: strings.Contains(systemTags, "owned"),
```

Comment:
There is a defined constant for "owned" that would be good to use here. Would you please replace the string here with the `OwnedAssetGroupTag` constant?